src/Observables.jl

```diff
@@ -130,6 +130,8 @@ end
 
 Adds function `f` as listener to `observable`. Whenever `observable`'s value
 is set via `observable[] = val` `f` is called with `val`.
+
+Returns `true` if `f` could be removed, otherwise `false`.
```

Comment:
This is incorrect?

Reply:
Is this correct?

Reply:
wow I stared at this for too long until I understood what was wrong :) good call

Reply:
Haha nice fix!